### ISSUE TITLE
Add support of messages edition

### ISSUE THREAD

**Issue by rbignon** [vector-im/riot-android#3465]
On Riot Web, it is possible to edit message.

When someone has edited his message, Riot Android prints a new message beginning with `* ` instead of change the original one.

**Comment by bmarty** [comment]
Yes, this will be available soon in Riot-Android. For the moment you can try to use RiotX, which is still in beta, but supports message edition
